test/unit/src/common/actions/t_snaps.js

```diff
@@ -21,10 +22,12 @@ const mockStore = configureMockStore(middlewares);
 
 describe('repositories actions', () => {
   const initialState = {
-    isFetching: false,
-    success: false,
-    error: null,
-    snaps: null
+    snaps: {
```

Comment:
Oh... so it seems we do it wrong in all the other actions tests, because it's not initial state of single reducer, but whole store...

And only because we don't do anything with initial state it doesn't break anything. Sounds like another technical debt to fix.